Modix.Data/Migrations/20180808181500_MessageLogBehaviourDefaults.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Modix.Data.Models;
+
+namespace Modix.Data.Migrations
+{
+    public partial class MessageLogBehaviourDefaults : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData("BehaviourConfigurations", new[] { "Category", "Key", "Value" }, new[] { BehaviourCategory.MessageLogging.ToString(), "LoggingChannelId", "380603419410432002" });
```

Comment:
This is following a similar pattern as in 20180719181318_Initial.cs#28 , of setting defaults based on our server. Since we don't currently have a way of modifying behavior configs outside of raw database queries, this is the only place the defaults can be set for the moment - however I am interested in revamping that. Will have to wait until core restructuring by Cisien, though

Reply:
Sounds like we're on the same page, then.